docs/api-docs/partner/staying-current.md

```diff
@@ -0,0 +1,45 @@
+# Staying Current with Platform Changes
+
+<div class="otp" id="no-index">
+
+### On this page
+
+- [General updates](#general-updates)
+- [Cornerstone theme updates](#cornerstone-theme-updates)
+- [API updates](#api-updates)
+- [Checkout SDK updates](#checkout-sdk-updates)
+
+</div> 
+
+
+At BigCommerce, we strive to offer the best ecommerce experience to our customers. Occasionally, we make updates to our platform to improve performance, add new features, or fix bugs. This article provides general information on how to stay current with the latest changes.
+
+## General updates
+
+For up-to-date information about the latest releases, new features, and bug fixes, we recommend setting up [a feed alert](https://developer.bigcommerce.com/changelog#feeds) for the [Developer Changelog](https://developer.bigcommerce.com/changelog). Additionally, you can gain more visibility into product updates by attending BigCommerce’s [Town Hall](https://support.bigcommerce.com/s/article/BigCommerce-Town-Halls) webinar programs and reading the [Product](https://support.bigcommerce.com/s/blog) and [Developer](https://medium.com/bigcommerce-developer-blog) blogs.
+
```

Comment:
@ektravel, Since you mentioned setting up feeds in the General Updates section, there's no need to repeat the link in subsequent sections.